Gemfile.base

```diff
@@ -88,6 +88,7 @@ gem '3scale_client', '~> 2.6.1', require: false
 gem 'analytics-ruby', require: false
 
 group :development, :test do
+  gem 'activerecord-nulldb-adapter'
```

Comment:
this was necessary to precompile the assets in the docker build phase. Since when we are building the image, the database does not exist and webpacker:compile needs the database up,

Reply:
Uhhhh that is weird
Probably worth a PR in webpacker
And if not then we need to fix our code